### PR TITLE
Camera item movement

### PR DESCRIPTION
**Pull request by bkontou**

### CODE REVIEW

File: Assets/MoveObject.cs [hunk 1]
```diff
@@ -81,85 +83,31 @@ void Update()
...
     {
+        // If its touching something on the bottom layer
+        if (transform.GetChild(0).GetComponent<Collider2D>().IsTouchingLayers(LayerMask.GetMask("Bottom")))
+        {
+            return;
+        }
+
         // Move to bottom
         gameObject.layer = LayerMask.NameToLayer("Bottom");
         transform.localScale *= (2 - elevation_scale_factor);
```
Comment (HBurd) [inline Assets/MoveObject.cs:113]: I think this is what causes the bug where things can decrease in size indefinitely, we can fix it by using absolute scales for top and bottom

File: Assets/MoveObject.cs [hunk 2]
```diff
@@ -195,14 +143,47 @@ public void SelectThisObject(Vector3 click_pos)
...
     {
         previous_object = selected_object;
+        // hide the weird little sprite i added to show that it was selected
+        selected_object.GetComponent<MoveObject>().selected_sprite.SetActive(false);
+        // Attempt to drop this object to the bottom layer
+        selected_object.GetComponent<MoveObject>().dropObject();
         selected_object = null;
     }
 
+    public void dropObject()
```
Comment (HBurd) [inline Assets/MoveObject.cs:159]: Do you need this function or can you just always call `BringToBottom`, since that also checks if something is below?

**Review by HBurd — COMMENTED** [review]
Also the sprite material issue should be fixed before merging

File: Assets/MoveObject.cs (hunk 2 above)
Comment (bkontou) [reply Assets/MoveObject.cs:159]: I guess not

**Comment by HBurd** [comment]
A few other things to fix and then this is good to merge:
- TMP warnings
- Object controls should be hidden at first because nothing is selected
- random plant spawned by default

**Pull request merged**